CONTRIBUTING.md

```diff
@@ -56,19 +56,19 @@ Please follow the next rules for the commit messages:
 ### Pull requests
 
 - We use [GitHub pull requests](https://help.github.com/articles/about-pull-requests) to accept the contributions.
-- Except for proposals (see next point), please, use [pull requests](https://github.com/Qiskit/openqasm/pulls) as is to submit a new one :smile:.
+- Except for proposals (see next point), please, use [pull requests](https://github.com/openqasm/openqasm/pulls) as is to submit a new one :smile:.
 - Review the parts of the documentation regarding the new changes and update it if it's needed.
 - New features often imply changes in the existent tests or new ones are needed. Once they're updated/added please be sure they keep passing.
 
 ## Spec proposals
 
-:bulb: All new ideas go through the next stages to become a new feature of the language.
+:bulb: All new ideas go through the following stages to become a new feature of the language:
 
-- Draft: Backlog items with different levels of abstraction. Anybody can add one issue in the [main repo](https://github.com/Qiskit/openqasm). Please label it as `draft`.
+- Draft: Backlog items with different levels of abstraction. Anybody can add an issue in the [main repo](https://github.com/openqasm/openqasm). Please label it as `draft`.
 - Proposal: An idea with the correct form:
-  - Add one issue in the [main repo](https://github.com/Qiskit/openqasm) labeled as `proposal` using [this template](templates/proposal.md).
+  - Add an issue in the [main repo](https://github.com/openqasm/openqasm) labeled as `proposal` using the spec proposal template.
```

Comment:
```suggestion
  - Add an issue in the [main repo](https://github.com/openqasm/openqasm) labeled as `proposal` using [the spec proposal template](https://github.com/openqasm/openqasm/issues/new?template=spec_proposal.yaml)
```

I _think_ but am not sure that this link should directly open the "new issue" dialogue with the new form.

Reply:
Thanks. I tested the link on my private GitHub trying things out repo and it works nicely.